components/arm/fake/fake.go

```diff
@@ -218,6 +218,10 @@ func (a *Arm) IsMoving(ctx context.Context) (bool, error) {
 	return false, nil
 }
 
+func (a *Arm) Kinematics(ctx context.Context) (referenceframe.Frame, error) {
+	return nil, errors.New("fake arm.Kinematics is unimplemented")
```

Comment:
Not sure if you are leaving unimplemented because you don't want to do this in this PR but in this case its super easy we just need to return `a.model`, utiizing the mutext appropriately of course